app/views/CategoryGuidanceView.scala.html

```diff
@@ -0,0 +1,42 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+layout: templates.Layout,
+govukButton: GovukButton,
+formHelper: FormWithCSRF,
+paragraph: components.Paragraph,
+heading1: components.Heading1
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(
+pageTitle    = titleNoForm(messages("profileSetup.title")),
+showBackLink = true
+) {
+@heading1(messages("categoryGuidance.h1"))
+@paragraph(Html(messages("categoryGuidance.p1")))
+@paragraph(Html(messages("categoryGuidance.p2")))
+@paragraph(Html(messages("categoryGuidance.p3")))
+
+@formHelper(action = CategoryGuidanceController.onSubmit) {
+@govukButton(
+ButtonViewModel(messages("site.continue"))
+)
+}
+}
+
```

Comment:
Should this be formatted? I know we don't have auto-formatter for view files but I think it'd be nice to format them manually anyway (not too much work to put in some indents). Up to disccus :)

Reply:
Other than this, looks great.

Reply:
done! :)